src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RATE = 40;
+  const DISCOUNT_MORE_THAN_7_DAYS = 50;
+  const DISCOUNT_MORE_THAN_3_DAYS = 20;
+  const RENTAL_DAYS_MORE_7 = 7;
+  const RENTAL_DAYS_MORE_3 = 3;
```

Comment:
The constants `RENTAL_DAYS_MORE_7` and `RENTAL_DAYS_MORE_3` could be named more clearly. Consider renaming them to `LONG_TERM_RENTAL_DAYS` and `SHORT_TERM_RENTAL_DAYS` respectively to better convey their purpose.